consensus/service/src/consensus_service.rs

```diff
@@ -704,24 +708,45 @@ impl<E: ConsensusEnclaveProxy, R: RaClient + Send + Sync + 'static> ConsensusSer
 
     /// Helper method for creating the get config json function needed by the GRPC admin service.
     fn create_get_config_json_fn(&self) -> GetConfigJsonFn {
+        let ledger_db = self.ledger_db.clone();
+        let byzantine_ledger = self.byzantine_ledger.clone();
         let config = self.config.clone();
         Arc::new(move || {
+            let block_height = ledger_db.num_blocks().unwrap_or(0);
```

Comment:
I think this should panic, or somehow be changed to report the error back. Otherwise, if this failed, executing `ledger_db.get_block(block_height - 1)` below is weird behavior.

Reply:
Reporting the error back makes sense, and can then display "null" for the block hash.